### PR TITLE
Fix/blocks

### PR DESCRIPTION
**Pull request by Ipomoea**

### CODE REVIEW

File: Sources/Functions/Block.swift [hunk 1]
```diff
@@ -23,14 +23,11 @@
 /// Closure with custom arguments and return value.
 public typealias Block<Input, Output> = (Input) -> Output
 
-/// Closure with no arguments and custom return value.
-public typealias ResultBlock<Output> = Block<Void, Output>
```
Comment (petropavel13) [inline Sources/Functions/Block.swift]: why removed?

File: Sources/Functions/Block.swift [hunk 2]
```diff
@@ -23,14 +23,11 @@
 /// Closure with custom arguments and return value.
 public typealias Block<Input, Output> = (Input) -> Output
```
Comment (petropavel13) [inline Sources/Functions/Block.swift]: maybe rename to `Closure<Input, Output>`? name VoidBlock came from legacy objc projects. we can keep `VoidBlock` as alias to () -> Void, but rename all others

File: Sources/Functions/Block.swift (hunk 1 above)
Comment (Ipomoea) [reply Sources/Functions/Block.swift]: because it doesn't work properly.
translates to this kind of code `... = (()) -> Output`

Comment (petropavel13) [reply Sources/Functions/Block.swift]: got the problem.
solution will be:
```swift
public typealias ResultClosure<Output> = () -> Output
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 0.9.11
+- **[Breaking change]**: Renamed `NumberFormat`'s `allOptions` to `allCases`
+- **Fix**: Closure syntax fix
```
Comment (petropavel13) [inline CHANGELOG.md]: don't forget to add new names

File: Sources/Functions/Block.swift [hunk 4]
```diff
@@ -21,19 +21,16 @@
...
-public typealias ResultBlock<Output> = Block<Void, Output>
+public typealias Closure<Input, Output> = (Input) -> Output
 
 /// Closure that takes custom arguments and returns Void.
-public typealias ParameterBlock<Input> = Block<Input, Void>
+public typealias ParameterClosure<Input> = Closure<Input, Void>
 
 /// Closure that takes no arguments and returns Void.
-public typealias VoidBlock = ResultBlock<Void>
+public typealias VoidBlock = () -> Void
```
Comment (petropavel13) [inline Sources/Functions/Block.swift]: ... and now ResultClosure can be moved back

**Pull request merged**